docs/pages/reference/mesheryctl/commands/mesheryctl_command_template.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: default
+title: Command Doc Title
+permalink: reference/mesheryctl/commands/template
+type: reference
+display-title: "false"
+language: en
+lang: en
+categories: en
+list: exclude
+# image: /assets/img/platforms/brew.png
+---
+<!-- 
+ This could be used as a template for commands documentation.
+ Just fill your information in the sections indicated below.
+ -->
+
+
+
+<!-- name of command -->
+# mesheryctl
+<pre class="codeblock-pre">
+ <div class="codeblock"><div class="clipboardjs">
+<!-- General example of command -->
+ mesheryctl &#60;command&#62; [flags]
+ </div></div>
+ </pre>
+
+<!-- General Synopse of command -->
+<!-- General Description of the command -->

```

Comment:
Could you please move this above the usage code block. i.e this should be right after the command heading.

```suggestion
<!-- Description of the command -->
```